src/Sulu/Bundle/MediaBundle/Tests/Features/collection_management.feature

```diff
@@ -42,6 +45,22 @@ Feature: Collection management
         And I am on the media collection edit page
         Then I expect to see "4" ".item" elements
 
+    Scenario: Move collection
+        Given the media collection "Dornbirn" exists
+        Given the media collection "Foobar" exists
+        Given I am editing the media collection "Dornbirn"
+        And I click toolbar item "collection-move"
+        And I expect an overlay to appear
+        And I expect the "husky.column-navigation.collection-select.initialized" event
+        And I click the column navigation item "Collections"
+        And I wait a second
+        And I double click the column navigation item "Foobar"
+        Given I am on "/admin/"
+        And I wait a second
```

Comment:
The "I am editing a media collection" should change the browser location. This seems strange.